### PR TITLE
DEV-292

### PR DESCRIPTION
**Pull request by levity**
Reorder fields in new user modal

### CODE REVIEW

**Review by thenerdcat — APPROVED** [review]
Changes pushed. I also had to remove height:18 in the helperBox class for fixing an issue with the margin of the modal input fields

**Comment by levity** [comment]
Going forward, you'll want to create different branches for different tasks so we don't accidentally merge work in progress

File: src/components/ApeTextField/ApeTextField.tsx [hunk 1]
```diff
@@ -71,7 +71,6 @@ const useStyles = makeStyles((theme) => ({
     padding: theme.spacing(0, 1),
   },
   helperBox: {
-    height: 18,
```
Comment (exrhizo) [inline src/components/ApeTextField/ApeTextField.tsx:74]: I had done this to get the create circle page to have it so when errors appeared, they didn't reflow the Ui, so if this is wierd in other places, I'd rather use the fixed height everywhere and adjust the spacing on the grid. We don't really have a unified standard for these types of things, something needing to work with our designer on.

Comment (exrhizo) [reply src/components/ApeTextField/ApeTextField.tsx:74]: Also these can all switch to the inline errors, which should work except for epoch has really long errors..

**Pull request merged**